crates/bevy_asset/src/asset_server.rs

```diff
@@ -216,6 +216,51 @@ impl AssetServer {
         self.load_untyped(path).typed()
     }
 
+    /// Create a new asset from another one
+    pub fn create_from<FROM: Asset, TO: Asset, F>(
```

Comment:
I like the use of descriptive generic parameters here but I find the mix of descriptive names (FROM) and one letter names (F) a bit hard to read. I think even just FN would make this flow better for me.

Also, should they be in CONSTANT_CASE or TypeCase? I feel like I've seen both. I thought the rust API guidelines had a suggestion on that but I can't find it so maybe I imagined it.

Reply:
upper camel case according to https://doc.rust-lang.org/rust-by-example/generics.html#generics

Reply:
Yeah, that's the documentation I was thinking of just not in the place I thought it would be.